list.php

```diff
@@ -1,159 +1,182 @@
-<?php 
-   /*
-  @Author : Mfsi_Annapurnaa
-  @purpose : handle thelisting of employee data.
-  : Deletion of a row
+<?php
+
+/**
+ * @Author : Mfsi_Annapurnaa
+ * @purpose : Handle listing of employee data and delete operation
  */
 
-require_once('config/dbConnect.php');
+require_once('config/session.php');
+require_once('config/queryOperation.php');
+
+$objSes = new session();
+$objSes->start();
+
+$resultSes = $objSes->checkSession();
+
+if (!$resultSes)
+{
+    header('Location:login.php');
+}
+
+$obj = new queryOperation();
+
 // Delete a row
 if (isset($_GET['delete']))
 {
     $empId = $_GET['delete'];
 
     // Extract image name and delete it
-    $imgQuery = "SELECT image FROM Employee WHERE Employee.empId = $empId";
-    $img = mysqli_fetch_array(mysqli_query($conn, $imgQuery));
-    unlink(IMAGEPATH . $img['image']);
-
-    $deleteAddress = "DELETE
-         FROM Address
-         WHERE Address.empId = $empId";
-    $deleteEmployee = "DELETE
-         FROM Employee
-         WHERE Employee.empId = $empId";
-
-    $delResultAddr = mysqli_query($conn, $deleteAddress);
-    $delResultEmp = mysqli_query($conn, $deleteEmployee);
+    $condition = ['column' => 'Employee.empId', 'operator' => '=', 'val' => $empId];
+    $result = $obj->select('Employee', 'image', $condition);
+    $img = mysqli_fetch_array($result);
 
-    if (!$delResultAddr && !$delResultEmp)
+    if (file_exists($img['image']))
```

Comment:
in this delete operation of deleting image/deleting employee record/deleting employee address, what if few operation failed and few executed.
better to executed all or none as all delete informations are done for employee information.